COMETwebapp/Components/OpenModel.razor

```diff
@@ -163,6 +171,9 @@ Copyright (c) 2022 RHEA System S.A.
             }
         }
 
+        this.isOpening = true;
+        StateHasChanged();
```

Comment:
should this have a `this.` qualifier? I dont see where the method is from

Reply:
It is a method from ComponentBase class, which informs that state has changed and the component needs to be rendered in the web page again.
Used to change button status during connection

Reply:
There is no `this.` qualifier because I don't need to re-write the existing function.